client/my-sites/site-settings/search.jsx

```diff
@@ -143,8 +152,38 @@ class Search extends Component {
 			isSearchModuleActive,
 			siteId,
 			translate,
+			saveJetpackSettings,
+			submitForm,
+			updateFields,
+			trackEvent,
 		} = this.props;
 
+		/**
+		 * Call WPCOM endpoints to update remote Jetpack sites' settings
+		 *
+		 * @param {boolean} jetpackSearchEnabled Whether Jetpack Search is enabled
+		 */
+		const handleInstantSearchToggleForJetpackSites = ( jetpackSearchEnabled ) =>
+			saveJetpackSettings( siteId, {
+				instant_search_enabled: jetpackSearchEnabled,
+			} );
+
+		/**
+		 * Change instant toggle status with Jetpack Search toggle and then save settings
+		 *
+		 * @param {boolean} jetpackSearchEnabled Whether Jetpack Search is enabled
+		 */
+		const handleJetpackSearchToggleForSimpleSites = ( jetpackSearchEnabled ) => {
+			trackEvent( `Toggled instant_search_enabled` );
```

Comment:
```suggestion
			trackEvent( 'Toggled instant_search_enabled' );
```

^ Switched to single quotes because we're not doing any variable interpolation.

Do we know where `trackEvent` records events? I see it used in the AMP component but I've never used it myself before.

Reply:
The toggle event was originally tracked in handleAutosavingToggle, since I changed not to use the function, I added the event tracking here. 
It seems it sends events to Google Analytics, but didn't see any requests sent out. Do you think I should look further?

Reply:
If we're just preserving the existing usage, that's all good :)